### PR TITLE
docs: Japanese translation (configure-api.md)

### PR DESCRIPTION
**Pull request by kumada626**
To translate `configure-api.md` to japanese.

### CODE REVIEW

**Review by yoshwata — APPROVED** [review]

**Comment by yoshwata** [comment]
[memo] 
It is not context of this PR. The link below might be better to change position.
[API config yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml#L268-L269)

File: docs/ja/cluster-management/configure-api.md [hunk 1]
```diff
@@ -486,11 +484,14 @@ scms:
             gheHost: github.screwdriver.cd # [Optional] Github Enterpriseの場合のGHEホスト
             username: sd-buildbot # [Optional] checkoutするユーザネーム
             email: dev-null@screwdriver.cd # [Optional] checkoutするユーザのEmailアドレス
+            commentUserToken: A_BOT_GITHUB_PERSONAL_ACCESS_TOKEN # [Optional] GithubでPRにコメントを書き込むためのトークン、"public_repo"のスコープが必要
```
Comment (wahapo) [inline docs/ja/cluster-management/configure-api.md]: I think that it is better to write `GithubのPR`.

File: docs/ja/cluster-management/configure-api.md [hunk 2]
```diff
@@ -486,11 +484,14 @@ scms:
...
             privateRepo: false # [Optional] プライベートレポジトリの read/write権限
 ```
 
 プライベートレポジトリを使用する場合は、`SCM_USERNAME` と `SCM_ACCESS_TOKEN` を [secrets](../../user-guide/configuration/secrets) として `screwdriver.yaml`に記述する必要があります。
 
+[メタPRコメント](../user-guide/metadata.md)を有効にするためには、Git上でbotユーザを作成し、そのユーザで`public_repo`のスコープを持ったトークンを作成する必要があります。Githubでは、新規にユーザを作成します。[create a personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line)の説明に従ってスコープを`public_repo`に設定します。このトークンをコピーして[API config yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml#L268-L269)内の`scms`の設定内の`commentUserToken`として設定します。
```
Comment (wahapo) [inline docs/ja/cluster-management/configure-api.md]: `Githubでは、新規にユーザを作成します。`→`Githubで新規にユーザーを作成し、`
In Japanese it seems more natural to connect with one sentence.

File: docs/ja/cluster-management/configure-api.md (hunk 1 above)
Comment (kumada626) [reply docs/ja/cluster-management/configure-api.md]: fixed.

File: docs/ja/cluster-management/configure-api.md (hunk 2 above)
Comment (kumada626) [reply docs/ja/cluster-management/configure-api.md]: fixed.

**Comment by kumada626** [comment]
I fixed translate corresponding review, and translate update configure-api.md which was updated by  https://github.com/screwdriver-cd/guide/pull/276 .

**Review by wahapo — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**